src/utils/content-to-jsx.tsx

```diff
@@ -52,6 +50,15 @@ export const contentToJsx = (content: Content, index?: number): JSXContent => {
       return <li key={index}>{contentToJsx(content.content)}</li>;
     case 'List':
       return content.order === 'Ascending' ? <ol key={index}>{contentToJsx(content.items)}</ol> : <ul key={index}>{contentToJsx(content.items)}</ul>;
+    case 'Claim':
+      return (
+        <section key={index}>
+          {(content.label || content.title) &&
+            <h4>{content.label && contentToJsx(content.label)} {content.title && contentToJsx(content.title)}</h4>
```

Comment:
If there is just title will there be a leading space?

Reply:
@E-Kemp Need some of your expert knowledge on React